images/libvirt/create-cluster

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
this entire file is out of place in `images` directory.

Also this seems like a release repo step-registry content.

Reply:
This file will be scp'd to the gcp instance where the libvirt install runs.  Why not keep it in this repository, since the installer team should be the ones who maintain it?

Reply:
I've been maintaining it for the past several months.

Reply:
these 2 files (provision-host and create-cluster) will be copied from the libvirt-installer image to the gcp instance launched through CI.